.github/workflows/build_package.yml

```diff
@@ -69,9 +69,9 @@ jobs:
       # Note that on Linux, we run under docker with an altered path.
       # Note that on Windows, we need to configure the compiler api project to
       # but its CMake build directory on a short path to avoid path overflow.
-      CIBW_ENVIRONMENT_LINUX: "REPO_DIR=/project/main_checkout BINDIST_DIR=/output CMAKE_GENERATOR=Ninja IREE_TARGET_BACKEND_CUDA=ON"
-      CIBW_ENVIRONMENT_MACOS: "REPO_DIR=${{ github.workspace }}/main_checkout CMAKE_GENERATOR=Ninja"
-      CIBW_ENVIRONMENT_WINDOWS: "REPO_DIR='${{ github.workspace }}\\main_checkout' CMAKE_GENERATOR=Ninja IREE_COMPILER_API_CMAKE_BUILD_DIR=D:/b"
+      CIBW_ENVIRONMENT_LINUX: "REPO_DIR=/project/main_checkout BINDIST_DIR=/output CMAKE_GENERATOR=Ninja IREE_TARGET_BACKEND_CUDA=ON IREE_HAL_DRIVER_CUDA=ON IREE_BUILD_TRACY=ON"
+      CIBW_ENVIRONMENT_MACOS: "REPO_DIR=${{ github.workspace }}/main_checkout CMAKE_GENERATOR=Ninja IREE_BUILD_TRACY=ON"
```

Comment:
can we also please add VULKAN ? That is all that is required for it to run on the GPU.

Reply:
I believe that is enabled by default? If not, can add it. Once this lands I'm going to plug my Mac in and give it a try.

Reply:
I think we still need it since VULKAN is not enabled by default on OSX. 

````
(base) anush@MacBook-Pro iree % cmake -G Ninja -B build/ -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ .                                                  
-- The ASM compiler identification is Clang
-- Found assembler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
-- The C compiler identification is AppleClang 13.1.6.13160021
-- The CXX compiler identification is AppleClang 13.1.6.13160021
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- IREE runtime drivers:
--   - dylib
--   - dylib-sync
--   - vmvx
--   - vmvx-sync
-- No build type selected, default to Release
-- Found Python3: /Users/anush/miniforge3/bin/python3.9 (found version "3.9.7") found components: Interpreter 
-- Looking for PyYAML - found
-- Found Git: /usr/bin/git (found version "2.32.0 (Apple Git-132)") 
-- Adding bundled LLVM source dependency
-- IREE compiler target backends:
--   - dylib-llvm-aot
--   - wasm-llvm-aot
--   - metal-spirv
--   - vulkan-spirv
--   - vmvx

````

vs with the flag

````
(base) anush@MacBook-Pro iree % cmake -G Ninja -B build/ -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DIREE_HAL_DRIVER_VULKAN=ON  .                     
-- IREE runtime drivers:
--   - dylib
--   - dylib-sync
--   - vmvx
--   - vmvx-sync
--   - vulkan
-- Found Python3: /Users/anush/miniforge3/bin/python3.9 (found version "3.9.7") found components: Interpreter 
-- Looking for PyYAML - found
-- Adding bundled LLVM source dependency
-- IREE compiler target backends:
--   - dylib-llvm-aot
--   - wasm-llvm-aot
--   - metal-spirv
--   - vulkan-spirv
--   - vmvx

````

Reply:
Ah, gotcha - because Vulkan is disabled by default on MacOS (I think it requires additional software installed on the host, right?). Ok, yes, will add it.

Reply:
Thanks for your patience. I am not a native MacOS user and just figuring this stuff out as I go.

Reply:
no no. thank you for this. Happy to help test. We have a lot of end users on OSX that this will make seamless.